post.hbs

```diff
@@ -22,7 +22,7 @@
         <header>
         {{#if tags}}<div class="post-meta tags">Posted in {{tags}}</div>{{/if}}
         <h1 class="post-title">{{{title}}}</h1>
-        <div class="post-meta"><a href="{{author.url}}">{{author.name}}</a> | <time datetime="{{date format='YYYY-MM-DD'}}">{{date format="dddd, MMMM DD, YYYY"}}</time></div>
+        <div class="post-meta">{{#author}}<a href="{{url}}">{{name}}</a>{{/author}} | <time datetime="{{date format='YYYY-MM-DD'}}">{{date format="dddd, MMMM DD, YYYY"}}</time></div>
```

Comment:
This puts the enclosed markup in the 'author' context, which is how `{{url}}` should work apparently.